test-gen-plugin/src/main/java/com/dylibso/chicory/maven/TestGenMojo.java

```diff
@@ -179,9 +179,9 @@ public void execute() throws MojoExecutionException {
                     includedExcludedWasts.add(includedWast);
                 }
             }
+            // TODO: this mechanism fails when there are no excluded wast in one profile only
```

Comment:
Nothing critical, but FYI @electrum , personally I think it's fine to relax those check as most of the spec have already been included and we will still review the incoming PRs.